modules/builds-understanding-openshift-pipeline.adoc

```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
 
 Pipelines give you control over building, deploying, and promoting your applications on {product-title}. Using a combination of the Jenkins Pipeline build strategy, `jenkinsfiles`, and the {product-title} Domain Specific Language (DSL) provided by the Jenkins Client Plug-in, you can create advanced build, test, deploy, and promote pipelines for any scenario.
 
-*{product-title} Jenkins Sync Plugin*
+*{product-title} Jenkins Sync Plug-in*
```

Comment:
@Preeticp, will you PTAL at the "plug-in" changes in this module? I can't find an official name for the plug-in, so I'm unsure if this capitalization change is correct.

Reply:
@Amrita42 please hold on this change. Rolfe felt we should make an effort to ask the Developers if we could change the name to say plug-in. If that is possible then your changes would be good and you would not need to revert.

Reply:
@Preeticp Thanks for reviewing. As discussed I will hold off till you get an affirmation from dev. I think it's great if we can keep plug-in in sync with our style standards. Thanks @rolfedh for making that effort.

Reply:
Hey @Amrita42, thanks to @rolfedh we have got acks from the devs to change it to the standard Plug-in, rather than Plugin. So your original changes can go in as is. Thank you, everyone, I'm happy we could agree on the standard way of writing the term.

Reply:
Here are the PRs created by Rolfe to change the upstream instances as well: https://github.com/openshift/jenkins-sync-plugin/pull/421/files and https://github.com/openshift/jenkins-client-plugin/pull/387/files (edited)

Reply:
Thank you everyone for this collabpration and I am glad we can have this consistency. Will update PR tday.

Reply:
I am resolving all my comments which are no longer relevant.